packages/liveblocks-core/src/room.ts

```diff
@@ -1713,7 +1718,8 @@ export function createRoom<
           }
 
           case ServerMsgCode.UPDATE_YDOC: {
-            eventHub.ydoc.notify(message.update);
+            const { isSync, update } = message;
+            eventHub.ydoc.notify({ update, isSync });
```

Comment:
Could we save the unpacking here, or did you do this deliberately?

```suggestion
            eventHub.ydoc.notify(message);
```